CHANGELOG.md

```diff
@@ -41,9 +41,12 @@ To install release candidates run `[sudo] gem install cocoapods --pre`
 * Do not merge `pod_target_xcconfig` from test specs into non test xcconfigs  
   [Dimitris Koutsogiorgas](https://github.com/dnkoutso)
   [#7037](https://github.com/CocoaPods/CocoaPods/pull/7037)
+* Fix globbing to traverse symlinks  
```

Comment:
nit: press enter

Reply:
Oops, think that got messed up during the rebase